exchanges/binance/binance_wrapper.go

```diff
@@ -636,6 +636,21 @@ func (b *Binance) UpdateAccountInfo(assetType asset.Item) (account.Holdings, err
 			})
 		}
 
+		acc.Currencies = currencyDetails
+	case asset.Margin:
+		accData, err := b.GetMarginAccount()
+		if err != nil {
+			return info, err
+		}
+		var currencyDetails []account.Balance
+		for i := range accData.UserAssets {
+			currencyDetails = append(currencyDetails, account.Balance{
+				CurrencyName: currency.NewCode(accData.UserAssets[i].Asset),
+				TotalValue:   accData.UserAssets[i].Free + accData.UserAssets[i].Borrowed,
```

Comment:
Last last last thing: Since this is the total it might be worth adding in LockedValue here

Reply:
After some rethinking, maybe this way?

```golang
currencyDetails = append(currencyDetails, account.Balance{
		CurrencyName: currency.NewCode(accData.UserAssets[i].Asset),
		TotalValue:   accData.UserAssets[i].Free + accData.UserAssets[i].Locked,
		Hold:         accData.UserAssets[i].NetAsset,
	})
```
> Disclaimer: I do not have margined trading activity so maybe my understanding is wrong

the `Free` might already include `Borrowed`, aka "Total asset = Free + Locked".
If `Hold` means "Net Asset", we simply plug that field in?

Reply:
Yep! That makes sense. At some point in the near future we need to create a separate field for Borrowed Assets for differentiability and keeping track of leverage(you don't have to worry about it in this PR unless it directly benefits you and you would like to)

Reply:
Commit made.